private/pkg/manifest/manifest.go

```diff
@@ -52,17 +52,9 @@ type Manifest struct {
 var _ encoding.TextMarshaler = (*Manifest)(nil)
 var _ encoding.TextUnmarshaler = (*Manifest)(nil)
 
-// New creates an empty manifest.
-func New() *Manifest {
```

Comment:
Attempting to make this usable with the zero value.

Reply:
This seems to do the trick. :)